docs/testing.md

```diff
@@ -1,5 +1,106 @@
 # Testing
 
+## Karma setup for Typescript
+
+Webpacker does not setup `Karma` by default, so you've to manually install it along with its dependencies as per your need. Following things marked as optional can be used to fancify the test results (Recommended).
+
+```js
+// package.json
+"scripts": {
+  "test": "NODE_ENV=test karma start"
+},
+"devDependencies": {
+  "typescript": "^2.5.2",
+  "ts-loader": "^2.3.7",
```

Comment:
Thanks for the other fix. Sorry, I wasn't very clear. I meant, `typescript` and `ts-loader` is a dependency (`dependencies`), where `karma` and other testing utils is for development only (`devDependencies`). Right?

Reply:
Yes. Moved them under dependencies.